src/Tribe/Views/V2/View.php

```diff
@@ -850,6 +853,9 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 				$url = add_query_arg( $query_args, $url );
 			}
 
+			// Remove the inverse of the page key we are using.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 1180f4a6692887c0d190ae319e912ae6</sub>